terminus-ui/autocomplete/src/autocomplete.component.ts

```diff
@@ -782,10 +793,8 @@ export class TsAutocompleteComponent implements OnInit,
    * @param selection - The item to select
    */
   public autocompleteSelectItem(selection: TsAutocompletePanelSelectedEvent): void {
-    if (!isString(selection.option.value)) {
-      throw Error('The value passing into autocomplete has to be string type');
-    }
-    const isDuplicate = this.autocompleteSelections.indexOf(selection.option.value) >= 0;
+    const isDuplicate = this.autocompleteSelections
+      .findIndex(s => selection.option.value === s.value) >= 0;
```

Comment:
do you always enforce `.value` attribution in option?

Reply:
as in requiring the user to specify a value on the `ts-option`?  good question, I can look to see if that's required